cirq/google/api/v1/programs.py

```diff
@@ -13,13 +13,13 @@
 # limitations under the License.
 import json
 from typing import (Any, cast, Dict, Iterable, Optional, Sequence, Tuple,
-                    TYPE_CHECKING, Union)
+                    TYPE_CHECKING)
 import numpy as np
 import sympy
 
 from cirq import devices, ops, protocols
 from cirq.schedules import Schedule, ScheduledOperation
-from cirq.value import Timestamp
+from cirq.value import Timestamp, type_alias
```

Comment:
Since you're not touching code in `cirq.value` I think it is preferable to put this up in the cirq imports: `from cirq import devices, ops, protocols, value`.  This also make the name `value.TParamVal` which is slightly nicer that `type_alias.TParamVal`.  

Note we've not been great about enforcing this pattern...even this file has some imports that I don't think should be done this way (all the classes).

Reply:
Done! I changed it for all instances.